[2025-06-03T13:39:15.965+0000] {processor.py:161} INFO - Started process (PID=166) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T13:39:15.969+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T13:39:15.970+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:39:15.970+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T13:39:15.981+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:39:15.979+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T13:39:15.982+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T13:39:16.085+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.130 seconds
[2025-06-03T13:39:47.244+0000] {processor.py:161} INFO - Started process (PID=177) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T13:39:47.247+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T13:39:47.250+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:39:47.248+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T13:39:47.265+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:39:47.262+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T13:39:47.265+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T13:39:47.328+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.101 seconds
[2025-06-03T13:40:17.867+0000] {processor.py:161} INFO - Started process (PID=187) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T13:40:17.870+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T13:40:17.873+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:40:17.872+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T13:40:17.881+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:40:17.879+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T13:40:17.881+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T13:40:17.925+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.090 seconds
[2025-06-03T13:40:48.336+0000] {processor.py:161} INFO - Started process (PID=197) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T13:40:48.338+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T13:40:48.339+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:40:48.339+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T13:40:48.348+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:40:48.345+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T13:40:48.349+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T13:40:48.411+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.096 seconds
[2025-06-03T13:41:18.922+0000] {processor.py:161} INFO - Started process (PID=207) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T13:41:18.923+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T13:41:18.925+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:41:18.925+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T13:41:18.933+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:41:18.930+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T13:41:18.935+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T13:41:18.977+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.063 seconds
[2025-06-03T13:41:49.370+0000] {processor.py:161} INFO - Started process (PID=217) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T13:41:49.371+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T13:41:49.373+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:41:49.372+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T13:41:49.386+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:41:49.380+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T13:41:49.386+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T13:41:49.436+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.072 seconds
[2025-06-03T13:42:19.950+0000] {processor.py:161} INFO - Started process (PID=227) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T13:42:19.951+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T13:42:19.952+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:42:19.952+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T13:42:19.968+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:42:19.961+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T13:42:19.969+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T13:42:20.017+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.085 seconds
[2025-06-03T13:42:50.471+0000] {processor.py:161} INFO - Started process (PID=237) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T13:42:50.472+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T13:42:50.473+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:42:50.473+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T13:42:50.487+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:42:50.480+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T13:42:50.488+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T13:42:50.535+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.070 seconds
[2025-06-03T13:43:21.000+0000] {processor.py:161} INFO - Started process (PID=247) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T13:43:21.002+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T13:43:21.006+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:43:21.005+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T13:43:21.022+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:43:21.020+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T13:43:21.022+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T13:43:21.067+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.074 seconds
[2025-06-03T13:43:51.525+0000] {processor.py:161} INFO - Started process (PID=257) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T13:43:51.526+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T13:43:51.528+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:43:51.528+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T13:43:51.547+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:43:51.544+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T13:43:51.548+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T13:43:51.602+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.091 seconds
[2025-06-03T13:44:22.140+0000] {processor.py:161} INFO - Started process (PID=267) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T13:44:22.141+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T13:44:22.143+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:44:22.142+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T13:44:22.153+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:44:22.150+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T13:44:22.154+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T13:44:22.185+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.051 seconds
[2025-06-03T13:44:52.600+0000] {processor.py:161} INFO - Started process (PID=277) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T13:44:52.601+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T13:44:52.603+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:44:52.602+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T13:44:52.609+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:44:52.608+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T13:44:52.610+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T13:44:52.643+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.050 seconds
[2025-06-03T13:45:23.121+0000] {processor.py:161} INFO - Started process (PID=287) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T13:45:23.122+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T13:45:23.124+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:45:23.123+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T13:45:23.140+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:45:23.136+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T13:45:23.140+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T13:45:23.175+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.061 seconds
[2025-06-03T13:45:53.674+0000] {processor.py:161} INFO - Started process (PID=297) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T13:45:53.675+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T13:45:53.677+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:45:53.676+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T13:45:53.690+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:45:53.685+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T13:45:53.690+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T13:45:53.732+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.068 seconds
[2025-06-03T13:46:24.186+0000] {processor.py:161} INFO - Started process (PID=307) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T13:46:24.187+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T13:46:24.188+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:46:24.188+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T13:46:24.200+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:46:24.198+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T13:46:24.200+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T13:46:24.236+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.065 seconds
[2025-06-03T13:46:54.735+0000] {processor.py:161} INFO - Started process (PID=317) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T13:46:54.739+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T13:46:54.741+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:46:54.740+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T13:46:54.748+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:46:54.747+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T13:46:54.751+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T13:46:54.794+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.068 seconds
[2025-06-03T13:47:25.281+0000] {processor.py:161} INFO - Started process (PID=327) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T13:47:25.284+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T13:47:25.285+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:47:25.285+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T13:47:25.292+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:47:25.291+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T13:47:25.293+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T13:47:25.332+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.060 seconds
[2025-06-03T13:47:55.755+0000] {processor.py:161} INFO - Started process (PID=337) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T13:47:55.756+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T13:47:55.758+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:47:55.757+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T13:47:55.773+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:47:55.770+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T13:47:55.773+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T13:47:55.819+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.072 seconds
[2025-06-03T13:48:26.249+0000] {processor.py:161} INFO - Started process (PID=347) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T13:48:26.255+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T13:48:26.258+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:48:26.257+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T13:48:26.275+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:48:26.271+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T13:48:26.276+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T13:48:26.313+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.078 seconds
[2025-06-03T13:48:56.719+0000] {processor.py:161} INFO - Started process (PID=357) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T13:48:56.721+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T13:48:56.722+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:48:56.721+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T13:48:56.728+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:48:56.727+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T13:48:56.729+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T13:48:56.783+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.071 seconds
[2025-06-03T13:49:27.281+0000] {processor.py:161} INFO - Started process (PID=367) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T13:49:27.287+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T13:49:27.292+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:49:27.291+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T13:49:27.306+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:49:27.304+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T13:49:27.306+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T13:49:27.351+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.078 seconds
[2025-06-03T13:49:57.883+0000] {processor.py:161} INFO - Started process (PID=377) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T13:49:57.885+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T13:49:57.887+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:49:57.886+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T13:49:57.897+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:49:57.894+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T13:49:57.899+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T13:49:57.935+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.074 seconds
[2025-06-03T13:50:28.443+0000] {processor.py:161} INFO - Started process (PID=387) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T13:50:28.447+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T13:50:28.450+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:50:28.449+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T13:50:28.459+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:50:28.458+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T13:50:28.460+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T13:50:28.535+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.109 seconds
[2025-06-03T13:50:59.114+0000] {processor.py:161} INFO - Started process (PID=397) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T13:50:59.115+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T13:50:59.117+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:50:59.116+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T13:50:59.133+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:50:59.129+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T13:50:59.134+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T13:50:59.166+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.060 seconds
[2025-06-03T13:51:29.653+0000] {processor.py:161} INFO - Started process (PID=407) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T13:51:29.656+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T13:51:29.658+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:51:29.657+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T13:51:29.673+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:51:29.669+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T13:51:29.674+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T13:51:29.717+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.070 seconds
[2025-06-03T13:52:00.220+0000] {processor.py:161} INFO - Started process (PID=417) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T13:52:00.222+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T13:52:00.225+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:52:00.224+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T13:52:00.246+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:52:00.244+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T13:52:00.247+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T13:52:00.296+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.082 seconds
[2025-06-03T13:52:30.712+0000] {processor.py:161} INFO - Started process (PID=427) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T13:52:30.713+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T13:52:30.714+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:52:30.714+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T13:52:30.729+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:52:30.723+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T13:52:30.729+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T13:52:30.776+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.084 seconds
[2025-06-03T13:53:01.225+0000] {processor.py:161} INFO - Started process (PID=437) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T13:53:01.226+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T13:53:01.227+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:53:01.226+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T13:53:01.245+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:53:01.234+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T13:53:01.245+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T13:53:01.297+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.078 seconds
[2025-06-03T13:53:31.752+0000] {processor.py:161} INFO - Started process (PID=447) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T13:53:31.754+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T13:53:31.755+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:53:31.755+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T13:53:31.774+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:53:31.767+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T13:53:31.774+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T13:53:31.807+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.079 seconds
[2025-06-03T13:54:02.343+0000] {processor.py:161} INFO - Started process (PID=457) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T13:54:02.344+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T13:54:02.346+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:54:02.345+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T13:54:02.353+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:54:02.352+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T13:54:02.354+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T13:54:02.404+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.070 seconds
[2025-06-03T13:54:32.906+0000] {processor.py:161} INFO - Started process (PID=467) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T13:54:32.907+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T13:54:32.908+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:54:32.908+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T13:54:32.924+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:54:32.917+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T13:54:32.926+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T13:54:32.981+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.081 seconds
[2025-06-03T13:55:03.586+0000] {processor.py:161} INFO - Started process (PID=477) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T13:55:03.594+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T13:55:03.598+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:55:03.596+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T13:55:03.612+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:55:03.609+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T13:55:03.614+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T13:55:03.649+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.076 seconds
[2025-06-03T13:55:34.105+0000] {processor.py:161} INFO - Started process (PID=487) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T13:55:34.121+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T13:55:34.136+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:55:34.133+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T13:55:34.164+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:55:34.160+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T13:55:34.166+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T13:55:34.267+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.209 seconds
[2025-06-03T13:56:04.847+0000] {processor.py:161} INFO - Started process (PID=497) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T13:56:04.853+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T13:56:04.857+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:56:04.856+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T13:56:04.880+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:56:04.873+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T13:56:04.881+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T13:56:04.941+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.113 seconds
[2025-06-03T13:56:35.410+0000] {processor.py:161} INFO - Started process (PID=507) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T13:56:35.411+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T13:56:35.412+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:56:35.412+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T13:56:35.429+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:56:35.421+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T13:56:35.430+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T13:56:35.478+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.081 seconds
[2025-06-03T13:57:05.908+0000] {processor.py:161} INFO - Started process (PID=517) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T13:57:05.912+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T13:57:05.913+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:57:05.913+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T13:57:05.924+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:57:05.919+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T13:57:05.924+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T13:57:05.957+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.066 seconds
[2025-06-03T13:57:36.424+0000] {processor.py:161} INFO - Started process (PID=527) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T13:57:36.428+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T13:57:36.431+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:57:36.431+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T13:57:36.449+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:57:36.444+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T13:57:36.450+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T13:57:36.492+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.083 seconds
[2025-06-03T13:58:06.998+0000] {processor.py:161} INFO - Started process (PID=537) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T13:58:06.999+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T13:58:07.001+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:58:07.000+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T13:58:07.008+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:58:07.006+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T13:58:07.009+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T13:58:07.044+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.054 seconds
[2025-06-03T13:58:37.582+0000] {processor.py:161} INFO - Started process (PID=547) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T13:58:37.590+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T13:58:37.600+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:58:37.596+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T13:58:37.625+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:58:37.622+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T13:58:37.625+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T13:58:37.720+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.166 seconds
[2025-06-03T13:59:08.172+0000] {processor.py:161} INFO - Started process (PID=557) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T13:59:08.184+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T13:59:08.186+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:59:08.185+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T13:59:08.194+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:59:08.192+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T13:59:08.194+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T13:59:08.244+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.092 seconds
[2025-06-03T13:59:38.754+0000] {processor.py:161} INFO - Started process (PID=567) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T13:59:38.761+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T13:59:38.762+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:59:38.762+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T13:59:38.770+0000] {logging_mixin.py:188} INFO - [2025-06-03T13:59:38.767+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T13:59:38.771+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T13:59:38.841+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.100 seconds
[2025-06-03T14:00:09.563+0000] {processor.py:161} INFO - Started process (PID=579) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T14:00:09.565+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T14:00:09.567+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:00:09.567+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T14:00:09.589+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:00:09.581+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T14:00:09.594+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T14:00:09.637+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.082 seconds
[2025-06-03T14:00:40.345+0000] {processor.py:161} INFO - Started process (PID=589) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T14:00:40.348+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T14:00:40.349+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:00:40.349+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T14:00:40.359+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:00:40.355+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T14:00:40.360+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T14:00:40.423+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.087 seconds
[2025-06-03T14:01:10.955+0000] {processor.py:161} INFO - Started process (PID=599) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T14:01:10.960+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T14:01:10.964+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:01:10.963+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T14:01:10.977+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:01:10.975+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T14:01:10.977+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T14:01:11.034+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.089 seconds
[2025-06-03T14:01:41.685+0000] {processor.py:161} INFO - Started process (PID=609) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T14:01:41.688+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T14:01:41.691+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:01:41.689+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T14:01:41.713+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:01:41.708+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T14:01:41.714+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T14:01:41.774+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.104 seconds
[2025-06-03T14:02:12.425+0000] {processor.py:161} INFO - Started process (PID=619) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T14:02:12.428+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T14:02:12.430+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:02:12.429+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T14:02:12.446+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:02:12.440+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T14:02:12.448+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T14:02:12.520+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.122 seconds
[2025-06-03T14:02:43.009+0000] {processor.py:161} INFO - Started process (PID=629) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T14:02:43.014+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T14:02:43.017+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:02:43.016+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T14:02:43.045+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:02:43.035+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T14:02:43.046+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T14:02:43.123+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.130 seconds
[2025-06-03T14:03:13.591+0000] {processor.py:161} INFO - Started process (PID=639) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T14:03:13.595+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T14:03:13.599+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:03:13.597+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T14:03:13.612+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:03:13.607+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T14:03:13.613+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T14:03:13.678+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.098 seconds
[2025-06-03T14:03:44.344+0000] {processor.py:161} INFO - Started process (PID=649) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T14:03:44.347+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T14:03:44.349+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:03:44.348+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T14:03:44.360+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:03:44.357+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T14:03:44.362+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T14:03:44.432+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.107 seconds
[2025-06-03T14:04:15.004+0000] {processor.py:161} INFO - Started process (PID=659) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T14:04:15.006+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T14:04:15.008+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:04:15.008+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T14:04:15.021+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:04:15.019+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T14:04:15.023+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T14:04:15.077+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.102 seconds
[2025-06-03T14:04:45.530+0000] {processor.py:161} INFO - Started process (PID=669) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T14:04:45.534+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T14:04:45.540+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:04:45.538+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T14:04:45.549+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:04:45.546+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T14:04:45.551+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T14:04:45.608+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.098 seconds
[2025-06-03T14:05:16.128+0000] {processor.py:161} INFO - Started process (PID=679) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T14:05:16.132+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T14:05:16.136+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:05:16.134+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T14:05:16.155+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:05:16.147+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T14:05:16.156+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T14:05:16.221+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.117 seconds
[2025-06-03T14:05:46.779+0000] {processor.py:161} INFO - Started process (PID=689) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T14:05:46.782+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T14:05:46.785+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:05:46.785+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T14:05:46.795+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:05:46.794+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T14:05:46.796+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T14:05:46.850+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.097 seconds
[2025-06-03T14:06:17.314+0000] {processor.py:161} INFO - Started process (PID=699) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T14:06:17.328+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T14:06:17.335+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:06:17.333+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T14:06:17.350+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:06:17.347+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T14:06:17.352+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T14:06:17.391+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.101 seconds
[2025-06-03T14:06:47.882+0000] {processor.py:161} INFO - Started process (PID=709) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T14:06:47.884+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T14:06:47.885+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:06:47.884+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T14:06:47.893+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:06:47.892+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T14:06:47.893+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T14:06:47.952+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.081 seconds
[2025-06-03T14:07:18.455+0000] {processor.py:161} INFO - Started process (PID=719) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T14:07:18.459+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T14:07:18.461+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:07:18.460+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T14:07:18.471+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:07:18.469+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T14:07:18.472+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T14:07:18.531+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.088 seconds
[2025-06-03T14:07:49.058+0000] {processor.py:161} INFO - Started process (PID=729) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T14:07:49.062+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T14:07:49.065+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:07:49.064+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T14:07:49.078+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:07:49.075+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T14:07:49.080+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T14:07:49.156+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.124 seconds
[2025-06-03T14:08:19.682+0000] {processor.py:161} INFO - Started process (PID=739) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T14:08:19.693+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T14:08:19.695+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:08:19.694+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T14:08:19.707+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:08:19.704+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T14:08:19.708+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T14:08:19.762+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.101 seconds
[2025-06-03T14:08:50.209+0000] {processor.py:161} INFO - Started process (PID=749) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T14:08:50.211+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T14:08:50.214+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:08:50.213+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T14:08:50.222+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:08:50.220+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T14:08:50.222+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T14:08:50.276+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.079 seconds
[2025-06-03T14:09:20.983+0000] {processor.py:161} INFO - Started process (PID=759) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T14:09:20.987+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T14:09:20.989+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:09:20.988+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T14:09:21.000+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:09:20.997+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T14:09:21.000+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T14:09:21.061+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.087 seconds
[2025-06-03T14:09:51.737+0000] {processor.py:161} INFO - Started process (PID=769) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T14:09:51.744+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T14:09:51.753+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:09:51.748+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T14:09:51.778+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:09:51.775+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T14:09:51.779+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T14:09:51.971+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.251 seconds
[2025-06-03T14:10:22.804+0000] {processor.py:161} INFO - Started process (PID=779) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T14:10:22.808+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T14:10:22.811+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:10:22.810+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T14:10:22.827+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:10:22.825+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T14:10:22.828+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T14:10:22.899+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.109 seconds
[2025-06-03T14:10:53.481+0000] {processor.py:161} INFO - Started process (PID=789) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T14:10:53.482+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T14:10:53.484+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:10:53.483+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T14:10:53.492+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:10:53.491+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T14:10:53.493+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T14:10:53.563+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.092 seconds
[2025-06-03T14:11:24.209+0000] {processor.py:161} INFO - Started process (PID=799) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T14:11:24.217+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T14:11:24.225+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:11:24.221+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T14:11:24.240+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:11:24.235+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T14:11:24.242+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T14:11:24.318+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.122 seconds
[2025-06-03T14:11:54.974+0000] {processor.py:161} INFO - Started process (PID=809) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T14:11:54.976+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T14:11:54.978+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:11:54.977+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T14:11:54.992+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:11:54.988+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T14:11:54.993+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T14:11:55.092+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.127 seconds
[2025-06-03T14:12:25.612+0000] {processor.py:161} INFO - Started process (PID=819) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T14:12:25.614+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T14:12:25.616+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:12:25.616+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T14:12:25.628+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:12:25.623+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T14:12:25.628+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T14:12:25.688+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.098 seconds
[2025-06-03T14:12:56.111+0000] {processor.py:161} INFO - Started process (PID=829) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T14:12:56.114+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T14:12:56.116+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:12:56.116+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T14:12:56.157+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:12:56.151+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T14:12:56.159+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T14:12:56.242+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.143 seconds
[2025-06-03T14:13:26.781+0000] {processor.py:161} INFO - Started process (PID=839) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T14:13:26.783+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T14:13:26.785+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:13:26.784+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T14:13:26.798+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:13:26.795+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T14:13:26.799+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T14:13:26.875+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.125 seconds
[2025-06-03T14:13:57.403+0000] {processor.py:161} INFO - Started process (PID=849) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T14:13:57.410+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T14:13:57.415+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:13:57.414+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T14:13:57.437+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:13:57.436+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T14:13:57.438+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T14:13:57.497+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.114 seconds
[2025-06-03T14:14:28.069+0000] {processor.py:161} INFO - Started process (PID=859) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T14:14:28.072+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T14:14:28.074+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:14:28.073+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T14:14:28.096+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:14:28.094+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T14:14:28.097+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T14:14:28.156+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.096 seconds
[2025-06-03T14:14:58.834+0000] {processor.py:161} INFO - Started process (PID=869) to work on /opt/airflow/dags/test_spark.py
[2025-06-03T14:14:58.840+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/test_spark.py for tasks to queue
[2025-06-03T14:14:58.844+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:14:58.842+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test_spark.py
[2025-06-03T14:14:58.876+0000] {logging_mixin.py:188} INFO - [2025-06-03T14:14:58.864+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/test_spark.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test_spark.py", line 2, in <module>
    from airflow.providers.apache.spark.operators.spark_submit import SparkSubmitOperator
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2025-06-03T14:14:58.876+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/test_spark.py
[2025-06-03T14:14:58.927+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/test_spark.py took 0.106 seconds
